bot.py

```diff
@@ -1073,6 +1145,20 @@ async def onDay(self, ctx):
         except Exception as ex:
             await self.bot.sendErrorToAuthor(ctx)
 
+    async def move_users_for_vote(self, town_info):
+        # get users in day channels other than Town Square
+        users = list()
+        for c in town_info.dayChannels:
+            if c != town_info.townSquare:
+                users.extend(c.members)
+
+        await town_info.controlChannel.send(f'Moving {len(users)} players from daytime channels to **{town_info.townSquare.name}**.')
```

Comment:
Note: I changed this from `ctx.send` to `town_info.controlChannel.send` so that I could use it without an incoming context.

In general I think this may just be a more reliable way to reuse code, is to use the actual channel rather than the context. After all, my vote timer does not have a context when it's time to call `!vote`.